Extensions/Physics2Behavior/tests/physics2runtimebehavior.spec.js

```diff
@@ -178,8 +181,12 @@ describe('Physics2RuntimeBehavior', () => {
 
       let hasBounced = false;
       let stepIndex = 0;
-      while (stepIndex < 10 && !hasBounced) {
-        runtimeScene.renderAndStep(1000 / fps);
+      // Seems like the events function is called even if hasBounce is true.
```

Comment:
Not sure I understand this comment, of course the events function is supposed to be called, that's normal no?